zio-cli/shared/src/main/scala/zio/cli/HelpDoc.scala

```diff
@@ -68,6 +68,142 @@ sealed trait HelpDoc { self =>
       case x                             => x
     }
 
+  def toHTML(): String = {
```

Comment:
Ah, because this is a pure function, can you drop the `()` after `toHtml`?

Reply:
Done :)